--- vtsspp/cpuevents.c.orig	2013-03-08 06:21:00.111635774 -0600
+++ vtsspp/cpuevents.c	2013-03-08 06:21:35.204247805 -0600
@@ -54,6 +54,7 @@
 #define IA32_PERFEVTSEL0     0x186
 #define IA32_PMC0            0x0c1
 
+#ifndef MSR_PKG_ENERGY_STATUS
 /// SNB power MSRs
 #define MSR_PKG_ENERGY_STATUS  0x611
 #define MSR_PP0_ENERGY_STATUS  0x639
@@ -64,6 +65,7 @@
 #define MSR_CORE_C3_RESIDENCY 0x3fc
 #define MSR_CORE_C6_RESIDENCY 0x3fd
 #define MSR_CORE_C7_RESIDENCY 0x3fe ///SNB
+#endif
 
 /* Knight family, KNC */
 #define KNX_CORE_PMC0                   0x20
